src/pkg/count/keys.go

```diff
@@ -1,42 +1,62 @@
 package count
 
-type key string
+type Key string
```

Comment:
Curious, what's the reason for exporting this?

Reply:
Hah, I don't entirely remember.  Maybe I used it?  If there's no exported use by the end of the pr chain I'll unexport it.